lint/rule_name.go

```diff
@@ -29,12 +30,41 @@ const nameSeparator string = "::"
 
 var ruleNameValidator = regexp.MustCompile("^([a-z0-9][a-z0-9-]*(::[a-z0-9][a-z0-9-]*)?)+$")
 
-// NewRuleName creates a RuleName from segments.
+// NewRuleName creates a RuleName from an AIP number and a unique name within
+// that AIP.
+func NewRuleName(aip int, name string) RuleName {
+	// Determine the group.
+	group := ""
+	if aip > 0 && aip < 1000 {
+		group = "core"
+	}
```

Comment:
Instead of burying into the code, can we create a function or global variable to determine the group? It will help us find the right place to add other groups, e.g., "cloud", "youtube" and so on.

Reply:
Done.